docs/toc.yml

```
@@ -675,11 +675,24 @@ items:
     - name: Pinnable task pane
       href: outlook/pinnable-taskpane.md
     - name: Add-ins for Outlook on mobile devices
-      href: outlook/outlook-mobile-addins.md
+      items:
+      - name: Overview
+        href: outlook/outlook-mobile-addins.md
+        displayName: outlook add-ins on mobile
```

I don't think you need to include "on" for these - maybe something like this so it's keywords...
```suggestion
        displayName: outlook, add-ins, mobile
```